src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -667,6 +668,62 @@ def create_application(client, display_name, homepage=None, identifier_uris=None
     return result
 
 
+def grant_application(cmd, identifier, appid, expires='1'):
+    if not appid:
```

Comment:
unnecessary, for not optional argument, CLI command parser will ensure users supply the values.

Reply:
Removed optional argument validation